pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -291,6 +297,34 @@ def connection_list(context, **options):
     context.execute_cmd(lambda: cmd_connection_list(context, options))
 
 
+@connection_group.command('set-default', cls=PywbemtoolsCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@click.argument('name', type=str, metavar='NAME', required=False)
+@click.option('--clear', is_flag=True, default=False,
+              help=u'Clear default connection name.')
+@click.option('-v', '--verify', is_flag=True, default=False,
+              help=u'Prompt user to verify change before changing the '
+              'default connection definition).')
+@add_options(help_option)
+@click.pass_obj
+def connection_set_default(context, name, **options):
+    """
+    Set a connection as the default connection.
+
+    Sets either the connection defined in the name argument as the default
+    current connection definition or, if there is no argument on the command
```

Comment:
This is the command line help text, and there the name argument is called "NAME".

Reply:
Fixed DONE